tests/unit_tests/udf_wrapper_params/token_classification/single_model_single_batch_complete.py

```diff
@@ -25,6 +25,27 @@ def udf_wrapper():
     def run(ctx: UDFContext):
         udf.run(ctx)
 
+def work_with_span_udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_transformers_extension.udfs.models.token_classification_udf import \
+        TokenClassificationUDF
+    from tests.unit_tests.udf_wrapper_params.token_classification. \
+        mock_sequence_tokenizer import MockSequenceTokenizer
+    from tests.unit_tests.udf_wrapper_params.token_classification.\
+        single_model_single_batch_complete import \
+        SingleModelSingleBatchComplete as params
+
+    udf = TokenClassificationUDF(
+        exa,
+        work_with_spans=True,
+        batch_size=params.batch_size,
+        pipeline=params.mock_pipeline,
+        base_model=params.mock_factory,
+        tokenizer=MockSequenceTokenizer)
+
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
 
 class SingleModelSingleBatchComplete:
```

Comment:
I think, we should write these tests directly using the Standalone Mock and not continue with these cursed tests

Reply:
will do in another pr